frames/src/main/java/com/checkout/frames/component/cardholdername/CardHolderNameComponentState.kt

```diff
@@ -0,0 +1,9 @@
+package com.checkout.frames.component.cardholdername
+
+import com.checkout.frames.component.base.InputComponentState
+
+internal data class CardHolderNameComponentState(
+    val inputState: InputComponentState
+) {
+    val cardHolderName = inputState.inputFieldState.text
```

Comment:
can this be a lazy value?
```suggestion
    val cardHolderName by lazy { inputState.inputFieldState.text }
```